app/controllers/tenants/projects_controller.rb

```diff
@@ -24,7 +24,12 @@ def create
 
   def update
     @project = Project.find(params[:id])
-    @project.update_attributes!(new_params)
+    @project.update_attributes(new_params)
+    if @project.valid?
+      flash[:notice] = "#{@project.title} Updated"
+    else
+      flash[:errors] = "Invalid Attributes"
+    end
```

Comment:
I think this can be better stated - remove line 27 and do if @project.update_attributes(new_params)

update_attributes should return a true when it is successful and a false when it is not.